app/src/main/AndroidManifest.xml

```diff
@@ -1,25 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="co.touchlab.kampkit.android">
-    <uses-permission android:name="android.permission.INTERNET"/>
+
+    <uses-permission android:name="android.permission.INTERNET" />
 
     <application
+        android:name=".MainApp"
```

Comment:
I think that this is the same as before
`android:name="co.touchlab.kampkit.android.MainApp"`, because it just tags it on to the package name on line 3. Is that correct?